src/components/Nodes/GethConfig/ConfigForm.jsx

```diff
@@ -46,9 +46,15 @@ class ConfigForm extends Component {
     }
   }
 
-  async getConfig() {
-    const config = await geth.getConfig()
-    this.setState({ config })
+  async setDefaultConfig() {
+    // Set default config if no config set
+    const { client } = this.props
+    const { config } = client
+    if (config) {
+      return
```

Comment:
nit: was a bit surprised `config` was `null`, rather than an object whose keys are null. I like that the reducer initial state can be good documentation for the data in the app. What do you think about having that default value contain the keys? e.g. `config: { host: null, port: null, ... }`

Reply:
for sure that sounds good, i can do that.
i did the same with `release` key so i can add the defaults there as well